app/furniture/section_navigation/section_grid_component.rb

```diff
@@ -33,7 +33,7 @@ def rooms
     end
 
     def section_images?
-      @section_images ||= rooms.any? { |r| r.hero_image.present? }
+      @section_images ||= rooms.any? { |r| r.hero_image? }
```

Comment:
Technically you can shrink this to `rooms.any?(&:hero_image?)` (but you don't need to)